src/DynamoCore/Graph/Nodes/CustomNodes/Function.cs

```diff
@@ -407,18 +407,6 @@ public Symbol(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts, T
             ElementResolver = elementResolver ?? new ElementResolver();
         }
 
-        /// <summary>
-        ///     Initializes a new instance of the <see cref="Symbol"/> class.
-        /// </summary>
-        [JsonConstructor]
```

Comment:
@sm6srw we NEED at least one constructor that is marked JSONConstructor.

Reply:
Fixed